tools/hardklor/hardklor.xml

```diff
@@ -0,0 +1,75 @@
+<tool id="hardklor" name="Hardklor" version="2.30">
+    <description>identification of features from mass spectra</description>
+    <requirements>
+        <requirement type="package" version="2.30">hardklor</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" level="fatal"/>
+    </stdio>
+    <command>python hardklor.py $ms1_in.extension $ms1_in
```

Comment:
I guess you need to use `$__tool_directory__/hardklor.py`

All params from type `data` and `text` should be quoted.